docs/containers-storage.conf.5.md

```diff
@@ -154,6 +154,10 @@ with the correct label.
 ## SEE ALSO
 `semanage(8)`, `restorecon(8)`
 
-# HISTORY
+## FILES
+
+Distributions often provide a /usr/share/containers/storage.conf file to define default storage configuration. Administrators usually can override this file by creating `/etc/containers/storage.conf` to specify their own configuration. The storage.conf file for rootless users is usually stored in the $HOME/.config/containers/storage.conf file.
```

Comment:
suggest s/Administrators usually can/Administrators can/

Reply:
And I'm tempted to suggest dropping 'usually' for the rootless users section too.